pkg/collector/metric/stats.go

```diff
@@ -84,12 +84,7 @@ func (s *UInt64Stat) SetNewAggr(newAggr uint64) error {
 		s.Curr = s.PrevCurr
 		return fmt.Errorf("Aggr value overflow %d < %d", newAggr, oldAggr)
 	}
-	if oldAggr == 0 {
-		// new value
-		s.Curr = 0
-	} else {
-		s.Curr = newAggr - oldAggr
-	}
```

Comment:
@sunya-ch PTAL here

Reply:
I have this because the first read aggregated number will be a big number. And the s.Curr-0 will notbe  what we expect.

Reply:
ok, this is left unchanged now